source/tokens/contracts/interfaces/IERC1155.sol

```diff
@@ -0,0 +1,29 @@
+pragma solidity ^0.5.0;
+
+import "openzeppelin-solidity/contracts/introspection/IERC165.sol";
+
+/**
+    @title ERC-1155 Multi Token Standard basic interface
+    @dev See https://eips.ethereum.org/EIPS/eip-1155
+ */
+contract IERC1155 is IERC165 {
```

Comment:
Why are we storing this in the repo not just importing it?

Reply:
ERC1155.sol uses it. Need it to write test cases in Swap to ensure the integration does work.